src/hotel-search/client.js

```diff
@@ -65,7 +65,21 @@ HotelSearchClient.prototype = {
     var isSearchEnd = response.done || this.poller.isLastPolling()
     this.merger.mergeResponse(response, isSearchEnd);  
     this.lastRatesCount = response.count;
-    this.responseSearch = response.search;
+    this.responseSearch = this.prepareResponseSearch(response);
+  },
+
+  prepareResponseSearch: function(response) {
+    var responseSearch = response.search;
+    var region = responseSearch.region;
+    if (region) {
+      var cities = [];
+      var staticCities = this.merger.__staticData.cities;
```

Comment:
Can we add a method in merger where we can fetch static cities data from here? something like https://github.com/wego/wego-sdk-js/blob/BE-821-hotel-search-by-region/src/hotel-search/Merger.js#L31

Else, let's make this `__staticData` public by changing it into `staticData` so that we can use it anywhere.

I prefer the first.